torch_em/transform/raw.py

```diff
@@ -227,3 +227,19 @@ def get_default_mean_teacher_augmentations(
         augmentation1=aug1,
         augmentation2=aug2
     )
+
+
+# The functionality below merges incoming and requested transforms to work together.
+class ConcatTransforms:
```

Comment:
This is not needed, we have https://github.com/constantinpape/torch-em/blob/main/torch_em/transform/generic.py#L36 already.